<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>270</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>1</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>402</width>
    <height>270</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>402</width>
    <height>270</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Download Manager</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="currentGroupBox">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>180</y>
      <width>116</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Grande</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>[Current Speed]</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QLCDNumber" name="currentSpeedLcdNum">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>22</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="digitCount">
      <number>7</number>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
    </widget>
    <widget class="QLabel" name="currentSpeedLabel">
     <property name="geometry">
      <rect>
       <x>73</x>
       <y>24</y>
       <width>38</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>KB/sec</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="averageGroupBox">
    <property name="geometry">
     <rect>
      <x>274</x>
      <y>180</y>
      <width>116</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Grande</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="title">
     <string>[Average Speed]</string>
    </property>
    <widget class="QLabel" name="averageSpeedLabel">
     <property name="geometry">
      <rect>
       <x>73</x>
       <y>24</y>
       <width>38</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Grande</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>KB/sec</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="averageSpeedLcdNum">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>22</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Grande</family>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="digitCount">
      <number>7</number>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="settingsGroupBox">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>100</y>
      <width>383</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Grande</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="title">
     <string>[Settings]</string>
    </property>
    <widget class="QPushButton" name="chooseFolderButton">
     <property name="geometry">
      <rect>
       <x>292</x>
       <y>22</y>
       <width>91</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Grande</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>Set folder</string>
     </property>
    </widget>
    <widget class="QPushButton" name="downloadButton">
     <property name="geometry">
      <rect>
       <x>292</x>
       <y>50</y>
       <width>91</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Grande</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>Download</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="urlLineEdit">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>54</y>
       <width>280</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Grande</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="folderLineEdit">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>26</y>
       <width>280</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Grande</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
    </widget>
    <zorder>chooseFolderButton</zorder>
    <zorder>downloadButton</zorder>
    <zorder>urlLineEdit</zorder>
    <zorder>folderLineEdit</zorder>
    <zorder>progressroupBox</zorder>
    <zorder>progressroupBox</zorder>
    <zorder>progressroupBox</zorder>
   </widget>
   <widget class="QGroupBox" name="timeFromStartGroupBox">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>0</y>
      <width>195</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Grande</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>[Time From Start]</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QLCDNumber" name="startSecLcdNum">
     <property name="geometry">
      <rect>
       <x>128</x>
       <y>22</y>
       <width>39</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="digitCount">
      <number>4</number>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
    </widget>
    <widget class="QLabel" name="startSecLabel">
     <property name="geometry">
      <rect>
       <x>173</x>
       <y>24</y>
       <width>19</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>sec</string>
     </property>
    </widget>
    <widget class="QLabel" name="startMinLabel">
     <property name="geometry">
      <rect>
       <x>106</x>
       <y>24</y>
       <width>19</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>min</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="startMinLcdNum">
     <property name="geometry">
      <rect>
       <x>61</x>
       <y>22</y>
       <width>39</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="digitCount">
      <number>4</number>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
    </widget>
    <widget class="QLabel" name="startHourLabel">
     <property name="geometry">
      <rect>
       <x>49</x>
       <y>24</y>
       <width>14</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>h</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="startHourLcdNum">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>22</y>
       <width>39</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="digitCount">
      <number>4</number>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="remainingTimeGroupBox">
    <property name="geometry">
     <rect>
      <x>261</x>
      <y>0</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Grande</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>[Remaining Time]</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="remainingMinLabel">
     <property name="geometry">
      <rect>
       <x>106</x>
       <y>24</y>
       <width>19</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>min</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="remainingMinLcdNum">
     <property name="geometry">
      <rect>
       <x>61</x>
       <y>22</y>
       <width>39</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="digitCount">
      <number>4</number>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
    </widget>
    <widget class="QLabel" name="remainingHourLabel">
     <property name="geometry">
      <rect>
       <x>49</x>
       <y>24</y>
       <width>14</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>h</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="remainingHourLcdNum">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>22</y>
       <width>39</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="digitCount">
      <number>4</number>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="receivedFileGroupBox">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>50</y>
      <width>94</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Grande</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>[Received]</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QLCDNumber" name="receivedFileLcdNum">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>22</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="smallDecimalPoint">
      <bool>false</bool>
     </property>
     <property name="digitCount">
      <number>7</number>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
    </widget>
    <widget class="QLabel" name="receivedFileLabel">
     <property name="geometry">
      <rect>
       <x>72</x>
       <y>24</y>
       <width>22</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>MB</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="fileSizeGroupBox">
    <property name="geometry">
     <rect>
      <x>202</x>
      <y>50</y>
      <width>94</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Grande</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>[File Size]</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QLCDNumber" name="fileSizeLcdNum">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>22</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="digitCount">
      <number>7</number>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
    </widget>
    <widget class="QLabel" name="fileSizeLabel">
     <property name="geometry">
      <rect>
       <x>72</x>
       <y>24</y>
       <width>22</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>MB</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="remainingFileGroupBox">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>50</y>
      <width>94</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Grande</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>[Remaining]</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QLCDNumber" name="remainingFileLcdNum">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>22</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="smallDecimalPoint">
      <bool>false</bool>
     </property>
     <property name="digitCount">
      <number>7</number>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
    </widget>
    <widget class="QLabel" name="remainingFileLabel">
     <property name="geometry">
      <rect>
       <x>72</x>
       <y>24</y>
       <width>22</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>MB</string>
     </property>
    </widget>
    <zorder>remainingFileLcdNum</zorder>
    <zorder>remainingFileLabel</zorder>
    <zorder>progressroupBox</zorder>
   </widget>
   <widget class="QGroupBox" name="percentageGroupBox">
    <property name="geometry">
     <rect>
      <x>298</x>
      <y>50</y>
      <width>94</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Grande</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>[Percentage]</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QLCDNumber" name="percentageLcdNum">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>22</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="digitCount">
      <number>7</number>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
    </widget>
    <widget class="QLabel" name="percentageLabel">
     <property name="geometry">
      <rect>
       <x>77</x>
       <y>24</y>
       <width>15</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>%</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>212</x>
      <y>10</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>&lt;&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="progressroupBox">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>180</y>
      <width>141</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Grande</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>[Progress]</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="readyLabel">
     <property name="geometry">
      <rect>
       <x>52</x>
       <y>24</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Grande</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Ready</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>402</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
